os/include/tinyara/security_level.h

```diff
@@ -0,0 +1,69 @@
+/****************************************************************************
+ *
+ * Copyright 2023 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+#include <tinyara/config.h>
+
+/********************************************************************************
+ * Pre-processor Definitions
+ ********************************************************************************/
+#define HIGH_SECURITY_LEVEL     "1"
+#define LOW_SECURITY_LEVEL      "0"
+
+/********************************************************************************
+ * Global Type Declarations
+ ********************************************************************************/
+
+/****************************************************************************
+ * Public Types
+ ****************************************************************************/
+
+/********************************************************************************
+ * Public Variables
+ ********************************************************************************/
+
+/****************************************************************************
+ * Inline Functions
+ ****************************************************************************/
+
+/********************************************************************************
+ * Public Function Prototypes
+ ********************************************************************************/
+#if defined(CONFIG_SECURITY_LEVEL)
+/****************************************************************************
+ * Name: set_security_level
+ *
+ * Description:
+ *   This function write the level of security (which is saved in secure storage)
```

Comment:
1. typo, writes
2. You wrote `the level of security` and the `security_level` for different usages.
   But in `get_security_level`, it says `the level of security`. What is correct?

Reply:
I missed the `_` in get_security_level. I'll add it.